packages/core/components/StatusMessage/StatusMessage.module.css

```diff
@@ -17,7 +17,7 @@
     display: none;
 }
 
-.centeringParent {
+.centering-parent {
```

Comment:
should this file change to match:
packages/core/components/StatusMessage/StatusMessage.module.css:.centeringParent {

Reply:
I'm not quite sure if I understand the suggestion

Reply:
I think Kevin is referencing that in the component it is "centeringParent" if so - the reason this is a different casing has to do with how the css module is loaded in. In JS/TS the css module names need to be the camel case version of whatever this case is (hyphen-case?)